Pixelbin/Pixelbin.csproj

```diff
@@ -1,32 +1,33 @@
 <Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <TargetFrameworks>netstandard2.1</TargetFrameworks>
-    <Authors>Pixelbin</Authors>
-    <PackageId>Pixelbin</PackageId>
-    <Version>1.0.1</Version>
-    <PackageLicenseFile>LICENSE</PackageLicenseFile>
-    <Description>Pixelbin Backend SDK for C# helps you integrate the core Pixelbin features with your application.</Description>
-    <Copyright>2022 Pixelbin</Copyright>
-    <PackageTags>Pixelbin</PackageTags>
-    <PackageIcon>Pixelbin.jpeg</PackageIcon>
-    <PackageProjectUrl>https://www.pixelbin.io/</PackageProjectUrl> 
-    <RepositoryUrl>https://github.com/pixelbin-dev/pixelbin-csharp-sdk.git</RepositoryUrl> 
-    <RepositoryType>github</RepositoryType>
-    <Nullable>enable</Nullable>
-  </PropertyGroup>
+    <PropertyGroup>
+        <TargetFrameworks>netstandard2.1</TargetFrameworks>
+        <Authors>Pixelbin</Authors>
+        <PackageId>Pixelbin</PackageId>
+        <Version>1.1.0</Version>
+        <PackageLicenseFile>LICENSE</PackageLicenseFile>
+        <Description>Pixelbin Backend SDK for C# helps you integrate the core Pixelbin features with
+            your application.</Description>
+        <Copyright>2022 Pixelbin</Copyright>
+        <PackageTags>Pixelbin</PackageTags>
+        <PackageIcon>Pixelbin.jpeg</PackageIcon>
+        <PackageProjectUrl>https://www.pixelbin.io/</PackageProjectUrl>
+        <RepositoryUrl>https://github.com/pixelbin-dev/pixelbin-csharp-sdk.git</RepositoryUrl>

```

Comment:
Do we need to update the repository url to https://github.com/pixelbin-io/pixelbin-csharp-sdk ?

Reply:
Repository URL has been updated.